docs/prow.md

```diff
@@ -0,0 +1,397 @@
+
+### For More Information
+
+- See the [E2E Testing documentation](../test/e2e/README.md) for details on running and writing E2E tests
+- See the [Pipelines documentation](pipelines.md) for information about the deployment pipeline system
+- See the [EV2 Deployment documentation](ev2-deployment.md) for details on EV2 deployment processes
+
+## Working with Prow Jobs
+
+### Triggering Presubmit Jobs
+
+To trigger optional presubmit jobs on a pull request, add a comment to the PR with the appropriate test command:
+
+```text
+/test e2e-parallel
+```
+
+To re-run a failed job:
+
+```text
+/retest
+```
+
+To re-run a specific job:
+
+```text
+/test <job-trigger>
+```
+
+### Viewing Job Results
+
+Job results are reported directly on pull requests as GitHub checks. You can:
+
+1. View the status of all jobs in the PR's "Checks" tab
+2. Click on a specific job to see detailed logs and test output
+3. Access the Prow dashboard for more detailed information and job history
+
+### Job Execution Environment
+
+All Prow jobs run on OpenShift clusters managed by the OpenShift CI infrastructure:
+
+- Jobs use Kubernetes for orchestration
+- Jobs run with ci-operator for standardized build and test workflows
+- Most jobs run on the build02 cluster, with some specialized jobs on build07 or build10
+
+### Modifying Prow Jobs
+
+Prow job definitions are maintained in the [openshift/release](https://github.com/openshift/release) repository, not in this repository. The actual job files are **generated** from configuration files, so you must edit the configs and regenerate.
+
+To modify or add jobs:
+
+1. Fork the [openshift/release](https://github.com/openshift/release) repository
+2. Edit configuration files in `ci-operator/config/Azure/ARO-HCP/`:
+   - `Azure-ARO-HCP-main.yaml` for presubmit and postsubmit jobs
+   - `Azure-ARO-HCP-main__periodic.yaml` for periodic jobs
+   - `Azure-ARO-HCP-main__image-updater.yaml` for image updater jobs
+3. Generate the job definitions by running:
+
+   ```bash
+   make ci-operator-config
+   make jobs
+   ```
+
+4. Submit a pull request to the openshift/release repository
+5. The OpenShift CI team will review and merge the changes
+
+> [!IMPORTANT]
+>
+> - The job files in `ci-operator/jobs/Azure/ARO-HCP/` are auto-generated - do not edit them directly
+> - Changes to Prow jobs require approval from the OpenShift CI team and must follow their contribution guidelines
+> - See the [README](https://github.com/openshift/release/blob/master/ci-operator/config/Azure/ARO-HCP/README.md) in the config directory for detailed instructions
+
+## Best Practices
+
+### For Developers
+
+1. **Always wait for required jobs**: The `images` and `frontend-simulation` jobs must pass before merging
+2. **Use optional jobs judiciously**: Trigger environment-specific E2E tests when your changes affect those environments
```

Comment:
This won't work: if you change something in RP codebase and manually trigger run in INT or Staging, the check doesn't test your changes in these environments. This could work for E2E test case code only. I suggest to drop this line completely, or restate again that requesting run in INT, Staging or Prod environments should not be done in most cases.